cmd/juju/storage/remove.go

```diff
@@ -140,7 +150,7 @@ func (c *removeStorageCommand) Run(ctx *cmd.Context) error {
 			anyFailed = true
 		}
 	}
-	if anyAttached {
+	if anyAttached && c.modelType != model.CAAS {
```

Comment:
do we need a message for CAAS model?

Reply:
we already print the failure above. this message is just to tell the user they can use --force, which is not applicable to caas models